spec/kitchen/driver/vagrant_spec.rb

```diff
@@ -1334,32 +1334,52 @@ def run_command(_cmd, options = {})
 
       it "adds a lines for createhd in :customize" do
         config[:customize] = {
-          :createhd => {
-            :filename => "./d1.vmdk",
-            :size => 10 * 1024,
-          },
+          :createhd => [
```

Comment:
What about adding a new test instead of replacing old one ? so that both syntaxes are covered